temporalio/worker/_activity.py

```diff
@@ -201,16 +201,15 @@ async def drain_poll_queue(self) -> None:
 
     # Only call this after run()/drain_poll_queue() have returned. This will not
     # raise an exception.
-    # TODO(dan): based on the comment above it looks like the intention may have been to use
-    # return_exceptions=True
     async def wait_all_completed(self) -> None:
         running_tasks = [v.task for v in self._running_activities.values() if v.task]
         if running_tasks:
-            await asyncio.gather(*running_tasks, return_exceptions=False)
+            await asyncio.gather(*running_tasks, return_exceptions=True)
```

Comment:
This makes sense to me (we don't want to return until all tasks are complete) except now this swallows exceptions. Has this logic been hit and is raising somehow today? I suspect we did mean to have this be `return_exceptions=True` originally, just want to confirm whether there is something observed precipitating this change.